python-sdk/docs/configurations.rst

```diff
@@ -124,8 +124,6 @@ or by updating Airflow's configuration
    [astro_sdk]
    auto_add_inlets_outlets = True
 
-.. _configure_native_fallback:
```

Comment:
<img width="718" alt="image" src="https://user-images.githubusercontent.com/8811558/208521807-3599fe22-86d9-4d89-ac95-4d3324ecec1a.png">

The docs was broken ! it had reference at other wrong place